starters/next-wordpress-starter/lib/getUrlPath.js

```diff
@@ -0,0 +1,4 @@
+export const getUrlPath = (url) => {
+  var rxPath = new RegExp("http[s]?:\\/\\/?[^\\/]+\\/(.*[/]{1}.*/.+.+$)");
```

Comment:
```suggestion
  const rxPath = new RegExp("http[s]?:\\/\\/?[^\\/]+\\/(.*[/]{1}.*/.+.+$)");
```
Not much reason to use `var` with ES6